### DISCUSSION TITLE
No Next js version is detected in your project

### DISCUSSION THREAD

**Discussion by haji09** [vercel/next.js#54092] [Help]
### Summary

when running npm run build in vs code it is all clear with no errors and when deploying in vercel this error is comming,
Error: No Next.js version could be detected in your project. Make sure `"next"` is installed in "dependencies" or "devDependencies"

Although next is installed and also it is there in dependencies.

### Additional information

```js
"dependencies": {
    "@headlessui/react": "^1.7.16",
    "@heroicons/react": "^2.0.18",
    "@paypal/react-paypal-js": "^8.1.1",
    "axios": "^1.4.0",
    "bcryptjs": "^2.4.3",
    "chart.js": "^4.3.3",
    "cloudinary": "^1.40.0",
    "js-cookie": "^3.0.5",
    "mongoose": "^7.4.3",
    "next": "13.4.16", <----------------------------------
    "next-auth": "^4.23.0",
    "react": "18.2.0",
    "react-chartjs-2": "^5.2.0",
    "react-dom": "18.2.0",
    "react-hook-form": "^7.45.4",
    "react-responsive-carousel": "^3.2.23",
    "react-toastify": "^9.1.3"
  },
```

**Comment by icyJoseph** [comment]
Show us your project file structure, from the root.

If the Next.js app is not at the root of the project, you have to specify it: https://vercel.com/docs/concepts/deployments/configure-a-build#root-directory
